src/commands/Moderation/unban.ts

```diff
@@ -2,46 +2,48 @@ import { GuildSettings, readSettings } from '#lib/database';
 import { LanguageKeys } from '#lib/i18n/languageKeys';
 import { ModerationCommand } from '#lib/moderation';
 import type { GuildMessage } from '#lib/types';
+import { getModeration, getSecurity } from '#utils/functions';
 import type { Unlock } from '#utils/moderationConstants';
 import { getImage } from '#utils/util';
 import { ApplyOptions } from '@sapphire/decorators';
+import { resolveKey } from '@sapphire/plugin-i18next';
 import type { ArgumentTypes } from '@sapphire/utilities';
 
 @ApplyOptions<ModerationCommand.Options>({
 	aliases: ['ub'],
 	description: LanguageKeys.Commands.Moderation.UnbanDescription,
 	extendedHelp: LanguageKeys.Commands.Moderation.UnbanExtended,
-	requiredMember: false,
-	permissions: ['BAN_MEMBERS']
+	requiredClientPermissions: ['BAN_MEMBERS'],
+	requiredMember: false
 })
 export class UserModerationCommand extends ModerationCommand {
 	public async prehandle(message: GuildMessage) {
-		const bans = await message.guild
-			.fetchBans()
+		const bans = await message.guild.bans
+			.fetch()
 			.then((result) => result.map((ban) => ban.user.id))
```

Comment:
Same gripe as before.

Reply:
This one isn't really better when changed because it would become
```ts
let bans: string[] | null = null;

try {
  const guildBans = await message.guild.bans.fetch;
  bans = guildBans.map((ban) => ban.user.id);
} catch {
  // noop
}
```

Which isn't really better tbh.

Reply:
`safeWrapPromise` is, tho.

Reply:
Added `safeWrapPromise`